src/routes/index.ts

```diff
@@ -102,7 +102,7 @@ router.get('/module/:moduleId', (req: express.Request, res: express.Respone) =>
 
 // API Calls for ModuleItem
 router.post('/module/:moduleId/addMouduleItem', (req: express.Request, res: express.Respone) => {
-  moduleItem.registerModuleItem(req, res);
+  Permission.checkAdministrator(req, res, moduleItem.registerModuleItem);
```

Comment:
Should only a administrator be allowed to insert a new moduleitem or also the teacher of the module?